src/main/scala/org/broadinstitute/hail/utils/package.scala

```diff
@@ -429,4 +432,14 @@ package object utils extends Logging
         split.getPath.getName
     }
   }
+
+  implicit def toRichJSONWritable[T](x: T)(implicit jw: JSONWriter[T]): RichJSONWritable[T] = new RichJSONWritable(x, jw)
```

Comment:
Up to you, though.  It's not exactly the same paradigm as the other rich classes.